events/artifacts/wc_historical_artifacts.txt

```diff
@@ -0,0 +1,154 @@
+﻿namespace = wc_historical_artifacts
+
+###### ### #
+## Dark Crystal
+## by Socratespanda
+###### ### ##
+
+#Give Out Dark Crystal after Facing Corruptist & Caster_Unit
+#Most likely to fire in the 583 start date versus the Horde invasion from Draenor
+wc_historical_artifacts.0200 = {
+	scope = combat_side
+	hidden = yes
+
+	trigger = {
+		#Make sure they have the DLC enabled
+		#has_dlc_feature = royal_court
+
+		#Conditions for finding the Dark Crystal; opponent must be Corruptist (orcish_fel), and have Casters MAAs
+		AND = {
+			#Check for Caster_Unit (the Dark Crystal comes from Warlocks)
+			enemy_side = {
+				has_maa_of_type = caster_unit
+			}
+			#Check for Corruptist (orcish_fel) since they'd have an evil crystal
+			enemy_side.side_commander = {
+				has_faith = faith:orcish_fel
+			}
+
+			#Make sure we don't handle Dark_Crystal out like its candy on Halloween.
+			NOT = { any_artifact = { has_variable = dark_crystal } }
+		}
+	}
+
+	#Trigger the event that awards the combat winner the Dark Crystal artifact.
+	immediate = {	
+
+		scope:combat_winner = {
+			trigger_event = wc_historical_artifacts.0201
+		}
+	}
+}
+
+#Generates the Dark Crystal. Fires from wc_historical_artifacts.0200
+#This can have some flavor or lore, and hopefully an option to turn it over to the Silver Hand or someone else for Piety gain.
+wc_historical_artifacts.0201 = {
+	type = character_event
+	title = wc_historical_artifacts.0201.title
+	desc = wc_historical_artifacts.0201.desc
+
+	theme = battle
+
+	immediate = {
+
+		#Creating the Dark Crystal and saving it as a scope to manipulate later
+		create_artifact_dark_crystal_effect = { OWNER = root }
+
+		scope:newly_created_artifact = {
+			save_scope_as = dark_crystal
+		}
+
+		cp:councillor_court_chaplain = {
+			save_scope_as = realm_priest
+		}
+
+		#Let's pick a random "evil" faith to give a mysterious NPC who tells you about the dark_crystal
+		random_religion_global = {
+
+			random_faith = {
+				limit = {
+					OR = {
+						has_doctrine = doctrine_shadow_magic_accepted
+						has_doctrine = doctrine_shadow_magic_approved	
+					}
+				}
+				
+				save_scope_as = dark_faith
+			}
+		}
+
+		#Finding an additional 1-2 people to consult with about dark magic 
+		# 1, Realm Priest
+		if = {
+			limit = {
+				exists = scope:realm_priest
+			}
+			scope:realm_priest = {
+				save_scope_as = dark_magic_consultant_1
+			}
+		}
+		#2, Someone Mysterious, using dark_faith assigned above
+		create_character = {
+			template = witchy_template
+			#faith = faith:burning_legion_religion
+			location = root.location
+			faith = scope:dark_faith
+			gender = female
+			save_scope_as = dark_magic_consultant_2
+		}
+	}
+
+	left_portrait = {
+		character = root
+		animation = war_over_win
+	}
+
+	right_portrait = {
```

Comment:
I think it will look better with lover_left_portrait

Also:
```
artifact = {
	target = scope:dark_crystal
	position = lower_center_portrait
}
```

will look good probably